htdocs/main.php

```diff
@@ -276,12 +276,12 @@ function tplFromRequest($param)
 }
 $tpl_data['userPerms']   = $realPerms;
 $tpl_data['studyParams'] = array(
-                            'useEDC'      => $config->getSetting('useEDC') ?
-        $config->getSetting('useEDC') : false,
-                            'useProband'  => $config->getSetting('useProband') ?
-        $config->getSetting('useProband') : false,
-                            'useFamilyID' => $config->getSetting('useFamilyID') ?
-        $config->getSetting('useFamilyID') : false,
+                            'useEDC'      => $config->getSetting('useEDC'),
+                            'useProband'  => $config->getSetting('useProband'),
+                            'useFamilyID' => $config->getSetting('useFamilyID'),
+                            'useConsent'  => $config->getSetting(
+                                'ConsentModule'
+                            )['useConsent'],
```

Comment:
Can't you just get the setting `useConsent` directly instead of grabbing its parent first?